crates/router/src/services/email/assets/invite.html

```diff
@@ -1,243 +1,225 @@
-            <tr>
-                <td class="spacer-sm" style="
-            -premailer-height: 20;
-            -premailer-width: 80%;
-            line-height: 10px;
-            margin: 0 auto;
-            padding: 0;
-          " width="100%"></td>
-            </tr>
-            <tr>
-                <td class="copy" style="
-            color: #666;
-            font-family: Roboto, Helvetica, Arial, san-serif;
-            font-size: 14px;
-            text-align: center;
-            line-height: 20px;
-            margin-top: 20px;
-            padding: 0;
-          " 20px!important; align="center">
-                    <br />
-                    You have received this email because your administrator has invited you as a new user on
-                    Hyperswitch.
-                    <br />
+              </tr>
+              <tr>
+                <td style="height: 40px"></td>
+              </tr>
+              <tr>
+                <td
+                  style="
+                    font-size: 2.02rem;
+                    font-weight: 600;
+                    line-height: 2.5rem;
+                    color: #111326;
+                    min-width: 550px;
+                  "
+                >
+                  Welcome to HyperSwitch!
                 </td>
-            </tr>
-            <tr>
-                <td class="spacer-sm" style="
-            -premailer-height: 20;
-            -premailer-width: 80%;
-            line-height: 10px;
-            margin: 0 auto;
-            padding: 0;
-          " width="100%"></td>
-            </tr>
-            <tr>
-                <td class="copy" style="
-            color: #666;
-            font-family: Roboto, Helvetica, Arial, san-serif;
-            font-size: 14px;
-            text-align: center;
-            line-height: 20px;
-            margin-top: 20px;
-            padding: 0;
-          " 20px!important; align="center">
-                    <br />
-                    <b>To get started, click on the button below. </b>
+              </tr>
+              <tr>
+                <td style="height: 10px"></td>
+              </tr>
+              <tr>
+                <td
+                  style="
+                    color: #666666;
+                    width: 50%;
+                    font-size: 1.02rem;
+                    font-weight: 400;
+                    line-height: 1.4rem;
+                    min-width: 300px;
+                  "
+                >
+                  <table
+                    style="
+                      width: 50%;
+                      min-width: 440px;
+                      text-align: center;
+                      margin: auto;
+                    "
+                  >
+                    <tr>
+                      <td>
+                        Hi {username}, You have received this email because your
+                        administrator has invited you as a new user on
+                        Hyperswitch.
```

Comment:
Sometimes we are using `Hyperswitch` and sometimes `HyperSwitch`. Can we make this consistent.